covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -107,7 +107,7 @@ def get_predictions(start_date_str, end_date_str, pres_df, countries=None):
 
     # Go to covid-xprize root dir to access predict script
     wd = os.getcwd()
-    os.chdir("../../..")
+    os.chdir("../../../..")
```

Comment:
Not sure about this one -- @EKMeyerson ? Actually I'm not keen on a Python script messing with the cwd in general -- maybe we can discuss a better way.